backend/capellacollab/projects/toolmodels/modelsources/git/exceptions.py

```diff
@@ -62,7 +62,7 @@ async def git_instance_api_endpoint_not_found_handler(
     return await exception_handlers.http_exception_handler(
         request,
         fastapi.HTTPException(
-            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            status_code=status.HTTP_422_UNPROCESSABLE_ENTITY,
```

Comment:
I'd say that `HTTP_400_BAD_REQUEST` is better here. The client is sending a request referencing an "invalid" Git instance. Unprocessable entity should also be fine.

Reply:
I would stick with unprocessable entity then. The request is semantically valid.